src/modules/network.cpp

```diff
@@ -111,6 +111,7 @@ namespace modules {
 
     auto upspeed = network->upspeed(m_udspeed_minwidth);
     auto downspeed = network->downspeed(m_udspeed_minwidth);
+    auto latency = network->latency();
```

Comment:
This would stall the update of the function by whatever the latency is. So a high latency connection would have much slower update rates for the network module.